### PR TITLE
Progress indicator

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

**Comment by axemonkey** [comment]
Superb. :+1:

File: cla_public/apps/checker/filters.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"Checker specific template filters"
+
+from cla_public.apps.base import base
+
```
Comment (andyhd) [inline cla_public/apps/checker/filters.py]: 2 blank lines needed here for PEP8 compliance

File: cla_public/apps/checker/views.py [hunk 2]
```diff
@@ -138,6 +139,14 @@ def triggered(v):
         fields = map(lambda (name, field): (field), fields)
         return fields
 
+    @property
+    def is_completed(self):
+        return session.get(self.form_class.__name__, {}).get('is_completed', False)
+
```
Comment (andyhd) [inline cla_public/apps/checker/views.py]: 2 blank lines here

**Comment by s-block** [comment]
Looks Good

**Pull request merged**